src/styles/index.scss

```diff
@@ -1,3 +1,131 @@
+@import './variables';
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $card--border--color;
+  border-radius: 5px;
+
+  &__image {
+    max-width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__img {
+    display: block;
+    width: 100%;
+    height: 134px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $black;
+    margin-bottom: 4px;
+  }
+
+  &__id {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $gray;
+    margin-bottom: 16px;
+  }
+
+  &-rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    &__reviews {
+      font-size: 10px;
+      font-weight: 400;
+      color: $black;
+      text-decoration: none;
+      line-height: 14px;
+      align-self: flex-end;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    .text {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $gray;
+    }
+
+    .value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $black;
+    }
+  }
+
+  &__link {
+    display: block;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $white;
+    background-color: $blue;
+    text-align: center;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    border-radius: 5px;
+    padding: 12px 70px;
+  }
+
+  &:hover {
+    .card__link {
+      background-color: $white;
+      color: $blue;
+      border: 1px solid $blue;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    display: block;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center center;
+    width: 16px;
+    height: 16px;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  &--4 {
```

Comment:
you should work not only with `--4` modifier but with 0,1,2,3,5 as well. Just copy your solution from `stars` task